### ISSUE TITLE
Use react-select for color ramp dropdowns

### ISSUE THREAD

**Issue by ShrimpCryptid** [allen-cell-animated/timelapse-colorizer#511]
## Use Case

Extension off of #408 ("Align dropdowns with WCAG rules (keyboard inputs") and #428 ("refactor: Use react-select for selection dropdowns"). Use `react-select` in the ColorRampDropdown for accessibility and UI consistency.

## Acceptance Criteria

- ColorRampDropdown should open only on click (not on hover) or spacebar button input.
- Up/down arrow keys should move the focused element in the list, and clicking again with the spacebar should select it and close the menu.
- Home/End should move to the top or bottom of the dropdown list.
- Styling/visual appearance should be identical to the original dropdowns.
